alternator/stats.cc

```diff
@@ -107,10 +120,14 @@ stats::stats() : api_operations{} {
                     seastar::metrics::description("total number of consumed write units, counted as half units"),{op("Index")})(alternator_label).set_skip_when_empty(),
             seastar::metrics::make_total_operations("filtered_rows_dropped_total", [this] { return cql_stats.filtered_rows_read_total - cql_stats.filtered_rows_matched_total; },
                     seastar::metrics::description("number of rows read and dropped during filtering operations"))(alternator_label).set_skip_when_empty(),
-            seastar::metrics::make_counter("batch_item_count", seastar::metrics::description("The total number of items processed across all batches"),{op("BatchWriteItem")},
+            seastar::metrics::make_counter("batch_item_count_total", seastar::metrics::description("The total number of items processed across all batches"),{op("BatchWriteItem")},
                     api_operations.batch_write_item_batch_total)(alternator_label).set_skip_when_empty(),
-            seastar::metrics::make_counter("batch_item_count", seastar::metrics::description("The total number of items processed across all batches"),{op("BatchGetItem")},
+            seastar::metrics::make_counter("batch_item_count_total", seastar::metrics::description("The total number of items processed across all batches"),{op("BatchGetItem")},
```

Comment:
The more I look at this, the more I'm confused... Why should the real count of items be called "batch_item_count_total" even though it's not a total - it's split into two labels - and the histogram as the same name without a "total"?

Maybe you can leave the existing batch_item_count with the same name (always a good idea for backward compatibility...) and call the new histogram batch_item_count_histogram? Or something?

Reply:
The suffix _total in Prometheus metrics means that it's accumulated (e.g. you probably need to derive it to make it meaningfully)